docs/proposals/compute-resource-metrics-api.md

```diff
@@ -35,15 +35,31 @@ Documentation for other releases can be found at
 
 ## Goals
 
-Provide resource usage metrics on pods and nodes on the API server to be used
-by the scheduler to improve job placement, utilization, etc. and by end users
-to understand the resource utilization of their jobs. Horizontal and vertical
-auto-scaling are also near-term uses.
+Provide resource usage metrics on pods and nodes through the API server to be
+used by the scheduler to improve job placement, utilization, etc. and by end
+users to understand the resource utilization of their jobs. Horizontal and
+vertical auto-scaling are also near-term uses. Additionally, a subset of the
```

Comment:
This comment has now been suppressed by the recent rebase :(

Yeah. I mean better documentation could satisfy my concerns. But in general, if we have a need for diverging the data format between the kubelet and api-server, we might have to abandon this idea. Since this is versioned and we are starting in an `alpha` phase, I'd say lets roll with the current proposal, with better documentation. WDYT?